openvdb_points/tools/PointDataGrid.h

```diff
@@ -439,12 +471,17 @@ class PointDataLeafNode : public tree::LeafNode<T, Log2Dim> {
         return totalPointCount;
     }
 
-    void deactivateEmptyVoxels()
+    /// @brief Uses the point count of each voxel to update the active voxel mask on this
+    /// leaf node. If @a deactivateEmptyVoxels is enabled, then voxels with a point count
+    /// of zero will be deactivated. If @a activateNonEmptyVoxels is enabled, then voxels
+    /// with a non-zero point count will be activated. By default, both of these are enabled.
+    void updateActiveVoxelMask(bool deactivateEmptyVoxels=true, bool activateNonEmptyVoxels=true)
```

Comment:
I don't really like the fact you can call this method like this updateActiveVoxelMask(false, false) and it becomes a no-op. Probably the deactivate case is much more important than the activate case, because if there are empty voxels activated, iteration will be marginally slower, but if there are occupied voxels deactivated, iteration will skip points that the user may be expecting to find.

Maybe we should look at where this method is typically being used, but I'm thinking maybe it might make sense not to include these bool options at all though. Perhaps updateActiveVoxels() is simple and useful enough by itself? The other option is to include an Enum like the point_masks::Active, but I'm just not sure it's worth the extra complexity.